gitpod-io/website
# docs: flexible workspace timeouts

## Description

Related https://github.com/gitpod-io/gitpod/pull/15815

**Do Not Merge Until the above PR is deployed** _(ETA: tomorrow)_ 

## Related Issue(s)



Fixes #3267 

## How to test

Open Preview, Read & Verify the following:

- [`/pricing`](https://deploy-preview-3268--gitpod-kumquat.netlify.app/pricing)

<img width="391" alt="image" src="https://user-images.githubusercontent.com/55068936/213354033-255f48ad-1c03-4ef9-96f1-4ce1abe7fa57.png">


- [`/docs/introduction/learn-gitpod/the-life-of-a-workspace#timeouts`](https://deploy-preview-3268--gitpod-kumquat.netlify.app/docs/introduction/learn-gitpod/the-life-of-a-workspace#timeouts)
- [`/docs/references/gitpod-cli#timeout`](https://deploy-preview-3268--gitpod-kumquat.netlify.app/docs/references/gitpod-cli#timeout)





## Release Notes



```release-note
docs: Flexible Workspace Timeouts
```


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3268"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
